GoogleDataTransport/GDTCORTests/Unit/GDTCORFlatFileStorageTest.m

```diff
@@ -778,6 +789,18 @@ - (void)testCheckForExpirations_WhenBatchWithNotExpiredEventsExpires {
   // 0.2. Wait for batch expiration.
   [[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:batchExpiresIn]];
 
+  // 0.3. Expect that the storage delegate will not be invoked because no
+  //      events will expire in this test.
+  GDTCORMetricsControllerFake *metricsController = [[GDTCORMetricsControllerFake alloc] init];
+  [GDTCORFlatFileStorage sharedInstance].delegate = metricsController;
+  XCTestExpectation *metricsControllerExpectation =
+      [self expectationWithDescription:@"metricsControllerExpectation"];
+  metricsControllerExpectation.inverted = YES;
```

Comment:
I haven't seen `inverted` before. Is that why the check is for a positive `events.count` when there should be no events?

Reply:
Not quite, `inverted` just means that the expectation should not be fulfilled.  It's useful for asserting that something _doesn't_ happen. The `events.count` assertion is irrelevant because that block should never be called (if the test is to pass)– I remove it.